src/main/java/no/sikt/nva/channelregister/ChannelRegister.java

```diff
@@ -135,14 +149,32 @@ public Optional<ErrorDetails> getChannelRegisterErrors(DublinCore dublinCore,
     }
 
     public String lookUpInChannelRegisterForPublisher(Record record, String customer) {
-        return Optional.ofNullable(record.getPublication())
+        return shouldLookUpInDegreePids(record, customer)
+            ? lookUpInDegreePids(customer)
+            :         Optional.ofNullable(record.getPublication())
                    .map(Publication::getPublicationContext)
                    .map(PublicationContext::getBragePublisher)
                    .map(ChannelRegister::formatValue)
                    .map(value -> lookUpInPublisher(value, customer))
                    .orElse(null);
     }
 
+    private String lookUpInDegreePids(String customer) {
+        var customerIssuingDegree =
+            customerIssuingDegrees.stream().filter(c -> c.getBrage().equals(customer)).findFirst().get();
+        return  customerIssuingDegree.getChannelRegistryPidProd();
```

Comment:
Legg merke til denne. Observerer at PID er lit for test og prod. Vet ikke om det vil være sant for alltid?